### PR TITLE
enh(3cx) Add a global counter

### PR DESCRIPTION
**Pull request by UrBnW**
Hi,

Let's add a 3CX global counter.
Feel free to modify if ever I did not use the proper method to add this.

Thx 👍

### CODE REVIEW

**Comment by garnier-quentin** [comment]
Please use new perfdata only

**Comment by UrBnW** [comment]
Thx for your review, should be OK now 👍

File: apps/voip/3cx/restapi/mode/extension.pm [hunk 1]
```diff
@@ -53,9 +53,21 @@ sub set_counters {
...
+        { name => 'global', type => 0 },
         { name => 'extension', type => 1, cb_prefix_output => 'prefix_service_output', message_multiple => 'All extensions are ok' }
     ];
 
+    $self->{maps_counters}->{global} = [
+        { label => 'count', nlabel => '3cx.extensions.count', display_ok => 0, set => {
+                key_values => [ { name => 'count' } ],
+                output_template => 'Extensions count : %d',
+                perfdatas => [
+                    { label => 'count', template => '%d', min => 0 }
```
Comment (garnier-quentin) [inline apps/voip/3cx/restapi/mode/extension.pm:65]: label is not useful. please remove it

Comment (UrBnW) [reply apps/voip/3cx/restapi/mode/extension.pm]: Removed !

**Pull request merged**